.gitlab-ci/vxworks7/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2018-current Jeffrey H. Johnson <trnsz@pobox.com>
```

Comment:
Didn't you recently add a PR that removed the 'current' syntax?

Reply:
Yes, I've not rebased this yet.  Still a WIP.